docs/design/fl-parallelism/README.md

```diff
@@ -105,3 +105,38 @@ Connection tracking overall state diagram.
 
 Real connection tracking code will not use FSM, because the state diagram is too
 simple and full-featured FSM will bring useless complexity in implementation.
+
+## Switch availability data propagation
+
+For monitoring purposes, we need to expose the list of FL / regions where the 
+switch is available to external(monitoring) tools. To keep switch data
+consistent we should not update it (switch data) anywhere outside of network
+topology. So we must propagate switch availability info from floodlightrouter
+into the network topology.
+
+We can mixin switch availability data into events proxied by floodlightrouter
```

Comment:
Probably `mixin` -> `mix in` if I get it correctly.

Reply:
replaced